pkg/kapp/resources/resource_filter.go

```diff
@@ -5,6 +5,7 @@ package resources
 
 import (
 	"encoding/json"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
nit: third party exports go into the next section

Reply:
done